ReportingServicesTools/ReportingServicesTools.psd1

```diff
@@ -22,7 +22,7 @@
     Copyright = '2016 Microsoft Corporation'
     
     # Description of the functionality provided by this module
-    Description = 'Provides extra functionality for SSRS (SQL Server Reporting Services).'
+    Description = 'Provides extra functionality for SSRS (SQL Server Reporting Services) and Power BI Report Server.  Includes the ability to deploy SSRS Reports, Datasets, and DataSources; as well as Power BI Reports.  For more information about the capabilities of this module, please visit the Project Site listed on the left navigation.'
```

Comment:
Maybe the "please visit the Project Site listed on the left navigation" should be removed as it would only make sense from the PowerShell gallery and not any other gallery or just printing the module description from the commandline

Reply:
I agree. If there's a hyperlink you can provide instead I think that'd be easier for someone to use to find more info...

Reply:
Great feedback, thanks for that!